frontend/components/shelter/pet/add/Form.tsx

```diff
@@ -19,8 +21,7 @@ import {
 } from "yup";
 import { FormSection } from "./FormComponents";
 
-const initialPet: Animal.Attributes = {
-	id: "test", //TODO: revert to "" after testing
+const initialPet: NewAnimal = {
 	shelterId: "test", //TODO: revert to "" after testing
```

Comment:
Please revert

Reply:
was thinking that it can be fixed together with the login context implementation. because the shelter ID is supposed to come from the user token

Reply:
I see ok we need to remember to change it back